src/network/asyncore_pollchoose.py

```diff
@@ -631,7 +704,11 @@ def connect(self, address):
             raise socket.error(err, errorcode[err])
 
     def accept(self):
-        # XXX can return either an address pair or None
+        """
+        Accept incoming connections
+
+        .. todo:: FIXME: can return either an address pair or None
```

Comment:
I'm not sure it's a FIXME, it could also be just a comment. However, it's not a big deal so I'm accepting this hunk.

Reply:
The todo was added to consider the fixme. I consider it now well considered. I've formatted it as part of the docstring.